apps/web/src/pages/templates/editor/TemplateEditorPage.tsx

```diff
@@ -106,7 +107,7 @@ export default function TemplateEditorPage() {
                     mr={20}
                     data-test-id="submit-btn"
                     loading={isLoading || isUpdateLoading}
-                    disabled={readonly || loadingEditTemplate || isLoading}
+                    disabled={readonly || loadingEditTemplate || isLoading || !isDirty}
```

Comment:
Should we add an error message to explain why it is disabled? For example, the user adds a subject in email, but no content. It is generally allowed and would not trigger our errors that we show on clicking submit. I'm not sure if its clear, wdyt?

Reply:
Like not just because we add the not dirty check but in general maybe...

Reply:
All good :) Thanks for checking